.travis.yml

```diff
@@ -1,9 +1,6 @@
 language: php
 php:
-  - '5.5'
-  - '5.6'
   - '7.2'
-  - nightly
```

Comment:
:thinking: losing forwards compatibility is an issue, though.

Reply:
I can put it back and run tests again. 
The issue was that one of the libs (cant recall which one) had fixed php 7 requirements. Therefore php 8 broke it.

Reply:
```
$ composer install
Loading composer repositories with package information
Updating dependencies (including require-dev)
Your requirements could not be resolved to an installable set of packages.
  Problem 1
    - Installation request for php-http/mock-client ^1.3 -> satisfiable by php-http/mock-client[1.3.0].
    - php-http/mock-client 1.3.0 requires php ^5.5 || ^7.0 -> your PHP version (8.0.0-dev) does not satisfy that requirement.
  Problem 2
    - php-http/guzzle6-adapter v2.0.1 requires php ^7.1 -> your PHP version (8.0.0-dev) does not satisfy that requirement.
    - php-http/guzzle6-adapter v2.0.0 requires php ^7.1 -> your PHP version (8.0.0-dev) does not satisfy that requirement.
    - Installation request for php-http/guzzle6-adapter ^2.0 -> satisfiable by php-http/guzzle6-adapter[v2.0.0, v2.0.1].
The command "composer install" failed and exited with 2 during .
```